src/server/services/Actions.ts

```diff
@@ -191,7 +191,9 @@ class Actions {
           `Can't send "${parsedAmount}", as balance is smaller "${faucetBalance}"`
         );
       }
-      const dripAmount = parsedAmount * 10 ** decimals;
+
+      const dripAmount = BigInt(parsedAmount * 10 ** decimals);
```

Comment:
This is not a valid fix:
If `parsedAmount * 10 ** decimals` > `9007199254740991` you will run into the issue again.

You should take the drip amount from the user as string (so that `9999999999999999` is also valid), make it a BigInt, then do the math using the BigInt lib:
- https://github.com/substack/node-bigint#muln
- https://github.com/substack/node-bigint#mpown

Note that you need to be vary with the conversion: if a user passes 1.42, you do not want to convert that to a BigInt first (that would become `1`) but first apply the network decimals to make it a BigInt such as `1420000000` in planks.